install/bootstrap_livecd_remote_iso.md

```diff
@@ -217,350 +239,350 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
+
+<a name="prepare_site_init"></a>
+#### 4.4 Prepare Site Init
+
+First, prepare a shim to faciliate going through the site-init guide:
+
+ ```bash
+ pit# mkdir -vp /mnt/pitdata
+ pit# mount -v --bind /var/www/ephemeral/ /mnt/pitdata/
+ ```
+
+Follow the procedures to [Prepare Site Init](prepare_site_init.md) directory for your system.
+
+Finally, cleanup the shim:
+ ```bash
+ # this uses rmdir to safely remove the directory, preventing accidentaly removals if one does not notice the umount command fail.
+ pit# umount /mnt/pitdata/
+ pit# rmdir /mnt/pitdata
+ ```
 
-1. Copy the service config files generated earlier by `csi config init` for DNSMasq, Metal
-   Basecamp (cloud-init), and Conman.
+<a name="bring---up-the-pit-services-and-validate-pit-health"></a>
+### 5. Bring-up the PIT Services and Validate PIT Health
 
-    1. Copy files (files only, `-r` is expressly not used).
+1. Set the same variables from the `csi config init` step from earlier, and then invoke "PIT init" to setup the PIT server for deploying NCNs.
+   > **`NOTE`** `pit-init` will re-run `csi config init`, copy all generated files into place, apply the CA patch, and finally restart daemons. This will also re-print the `metalid.sh` content incase it was skipped in the previous step. **Re-installs** can skip running `csi config init` entirely and simply run `pit-init.sh` after gathering CSI input files into `/var/www/ephemeral/prep`.
 
-        ```bash
-        pit# cp -pv /var/www/ephemeral/prep/${SYSTEM_NAME}/dnsmasq.d/* /etc/dnsmasq.d/
-        pit# cp -pv /var/www/ephemeral/prep/${SYSTEM_NAME}/conman.conf /etc/conman.conf
-        pit# cp -pv /var/www/ephemeral/prep/${SYSTEM_NAME}/basecamp/* /var/www/ephemeral/configs/
-        ```
+    ```bash
+    pit# export SYSTEM_NAME=eniac
+    pit# export USERNAME=root
+    pit# export IPMI_PASSWORD=changeme
+    pit# /root/bin/pit-init.sh
+    ```
+
+1. Download and install/upgrade the workaround and documentation RPMs.
 
-    1. Enable, and fully restart all PIT services.
+   If this machine does not have direct Internet access these RPMs will need to be externally downloaded and then copied to the system.
 
-        ```bash
-        pit# systemctl enable basecamp nexus dnsmasq conman
-        pit# systemctl stop basecamp nexus dnsmasq conman
-        pit# systemctl start basecamp nexus dnsmasq conman
-        ```
+   **Important:** In an earlier step, the CSM release plus any patches, workarounds, or hotfixes
+   were downloaded to a system using the instructions in [Check for Latest Workarounds and Documentation Updates](../update_product_stream/index.md#workarounds). Use that set of RPMs rather than downloading again.
+
+   ```bash
+   pit# rpm -Uvh --force https://storage.googleapis.com/csm-release-public/shasta-1.5/docs-csm/docs-csm-latest.noarch.rpm
+   pit# rpm -Uvh --force https://storage.googleapis.com/csm-release-public/shasta-1.5/csm-install-workarounds/csm-install-workarounds-latest.noarch.rpm
```

Comment:
We have steps earlier on that apply workarounds.   I don't know if we are actually putting workaround in there anymore.  If not, we should remove all references to this. If so, this should be earlier.